README.md

```diff
@@ -306,6 +307,8 @@ similar to those available for Certificate Transparency logs
    information for particular leaves, specified either by their hash value or
    index in the log.
  - `QueueLeaves` requests inclusion of specified items into the log.
+     - For a pre-ordered log, `AddSequencedLeaves` requests the inclusion of
+       specified items into the log at specified places in the tree.
  - `GetInclusionProof`, `GetInclusionProofByHash` and `GetConsistencyProof`
     return inclusion and consistency proof data.
 
```

Comment:
nit: Might be worth re-wording the following paragraph a little, to make it clear that the signer is required for pre-ordered logs, even though you never call `QueueLeaves` (which could be interpreted as meaning there are never "pending queued items").

Reply:
Done.